typescript-sdk/src/clients/dispatcher/dispatcher-client.test.ts

```diff
@@ -39,6 +41,8 @@ describe('DispatcherClient', () => {
           ca_file: 'TLS_ROOT_CA_FILE',
           server_name: 'TLS_SERVER_NAME',
         },
+        api_url: 'API_URL',
+        tenant_id: 'tenantId',
```

Comment:
note for future: we should probably put the config in a root fixtures dir or something bc its a pain to propagate these changes